test/integration/features/addon-xpaas.feature

```diff
@@ -30,6 +30,7 @@ which are then available in OpenShift to the user.
     | template-name           | service-name   | http-endpoint | expected-hello                        |
     | datagrid65-basic        | datagrid-app   | /             | Welcome to the JBoss Data Grid Server |
     | eap70-basic-s2i         | eap-app        | /index.jsf    | Welcome to JBoss!                     |
+    | eap71-tx-recovery-s2i   | eap-app        | /index.jsf    | Welcome to JBoss!                     |
```

Comment:
I can see `eap71-basic-s2i` too in the templates. So is there any specific reason why we are using `eap71-tx-recovery-s2i` ?

Reply:
```
$ oc get templates -n openshift | grep eap71-basic
eap71-basic-s2i                                 An example EAP 7 application. For more information about using this template,...   16 (5 blank)      6
```

Reply:
Nothing specific, Just for a change in the sequence. As we have eap7.0, thought lets check with a different eap71-* (atleast a non basic-s2i)